system/internal/auth/social/main.go

```diff
@@ -2,8 +2,9 @@ package social
 
 import (
 	"github.com/crusttech/crust/internal/config"
+	"github.com/crusttech/crust/internal/settings"
 )
 
-func Init(c *config.Social) {
-	initGoth(c)
+func Init(c *config.Social, finder settings.Finder) {
```

Comment:
You shouldn't use `main.go` as the filename outside of cmd/ (use `init.go` or `setup.go` or something more apt)

Reply:
ack.